controlpanel/api/rules.py

```diff
@@ -49,8 +52,8 @@ def is_app_admin(user, obj):
 add_perm('api.destroy_app', is_authenticated & is_superuser)
 add_perm('api.add_app_customer', is_authenticated & is_app_admin)
 add_perm('api.remove_app_customer', is_authenticated & is_app_admin)
-add_perm('api.add_app_admin', is_authenticated & is_superuser)
-add_perm('api.revoke_app_admin', is_authenticated & is_superuser)
+add_perm('api.add_app_admin', is_authenticated & is_app_admin)
```

Comment:
This makes sense to me.

I'm trying to wrap my head around whether we could end up in the situation where a user Alice could be admin of the App "Foo" but not have access (whether is admin or not) for the corresponding app bucket "Foo" and whether that would be a problem or not. Maybe not. 

Maybe the granting admin will also have to grant permission to the app bucket as well if the wants to give access to the data. It may not be a bad thing as it would be more _explicit_ and potentially in the future we may have users who can admin an app (e.g. deploy/restart it or something) but not have access to the data. I don't know, I'm thinking "loudly" here.

I think the permission to grant access to the data is called `api.grant_s3bucket_access` and bucket admins can already grant permissions to other users.

Reply:
(sorry, stream of consciousness comment 🤣 )

Reply:
https://www.osnews.com/story/19266/wtfsm/ :+1: :wink:

Reply:
I think it's sensible to have separate administrative cover for app and bucket - there could conceivably be situations where someone needs to be in charge of the one but not the other.

Reply:
@samtazzyman yeah that was also the conclusion I reached by talking with myself 🤣

Reply:
OK... so AFAICT, given the `api.grant_s3bucket_access` permission and the change in this PR, we're covered in terms of the situation as described by @samtazzyman..? (I'm painfully aware that I don't have the org context, depth of knowledge and this is my first PR so I don't want to make assumptions). :+1: Thanks!

Reply:
@ntoll yeah bucket admins can already `api.grant_s3bucket_access` and you tweaked the permission to `api.add_app_admin` so I think your original changes should be enough once tests are also fixed/adjusted ☺️

Reply:
> once tests are also fixed/adjusted

Easier said than done..! :wink:

Reply:
Sorry 😁!! The medium doesn't help, didn't mean to be pushy or put pressure on you of course ☺️